update-dois/src/main/java/org/reactome/release/updateDOIs/UpdateDOIs.java

```diff
@@ -49,10 +56,13 @@ public static void findAndUpdateDOIs(long authorIdTR, long authorIdGK, String pa
 			instanceEditGK = UpdateDOIs.createInstanceEdit(UpdateDOIs.dbaGkCentral, authorIdGK, creatorFile);
 		}
 		// Gets the updated report file if it was provided for this release
-		Map<String, Map<String,String>> expectedUpdatedDOIs = UpdateDOIs.getExpectedUpdatedDOIs(pathToReport);
+		Map<String, Map<String,String>> expectedUpdatedDOIs = new HashMap<>();
+		File reportFile = new File(pathToReport);
+		if (reportFile.exists()) {
```

Comment:
You could use the `Files.exists()` method instead:

`if (Files.exists(Paths.get(pathToReport)))`

https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#exists-java.nio.file.Path-java.nio.file.LinkOption...-